source/CardGames.py

```diff
@@ -0,0 +1,201 @@
+        index += 1
+    
+    self.cards = deck
+    self.size = len(deck)
+    self.cardBack = cardBack
+    self.discarded = []
+
+  def __str__(self):
+      return (cardImages)
+
+  def shuffle(self):
+    random.shuffle(self.cards)
+
+  def getCard(self):
+    card = self.cards.pop()
+    self.size -= 1
+    self.discarded.append(card)
+    return card
+
+class Player:
+  def __init__(self, name, money: int = 0):
+    self.name = name
+    self.hand = []
+    self.knownCards = []
+    self.money = money
+
+  def addMoney(self, amount: int):
+    self.money += amount
+    return self.money
+
+  def makeBet(self, amount: int):
+    if amount > self.money:
+      print("%s does not have enough money to make this bet." % self.name)
+      return self.money
+    self.money -= amount
+    return self.money
+
+  def addCard(self, card: Card, isKnown: bool = True):
+    self.hand.append(card)
+    if isKnown:
+      self.knownCards.append(True)
+    else:
+      self.knownCards.append(False)
+
+  def clearHand(self):
+    self.hand = []
+    self.knownCards = []
+
+  def showHand(self):
+    print(self.hand)
+
+PlayerList = List[Player]
+
+class Dealer:
+  def __init__(self):
+    pass
+
+  def printCards(self, cards: CardList, showFront: bool, printShort: bool = True):
+    for idx in range(6):
+      for i, card in enumerate(cards):
+        if printShort and i < len(cards)-1:
+          image = card.shortImage[idx] if showFront else card.cardBack[idx]
+          print(image, end="")
+        else:
+          image = card.image[idx] if showFront else card.cardBack[idx]
+          print(image, end="")
+      print()
+    
+  def printPlayerCards(self, player: Player, printShort: bool = False):
+    for idx in range(6):
+      for i, card in enumerate(player.hand):
+        if printShort and i < len(player.hand)-1:
+          image = card.shortImage[idx]  if player.knownCards[i] else card.cardBack[idx]
+          print(image, end="")
+        else:
+          image = card.image[idx] if player.knownCards[i] else card.cardBack[idx]
+          print(image, end="")
+      print()
+
+  def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
+    if numCards * len(players) > deck.size:
+      return False
+    for player in players:
+      for _ in range(numCards):
+        player.addCard(deck.getCard())
+    return True
+
+
+  
+# NEW CODE BELOW
+
+  def printAllPlayerCards_test(self, players: PlayerList):
+  #  TotalPlayers = []
+    for i in players:
+      self.printPlayerCards(i)
+
+
+# COMMENTED OUT SETUP CODE      
+"""
+deck1 = Deck()
+player1 = Player("john")
+player2 = Player("jeff")
+player3 = Player("george")
+players2 = [player1, player2, player3]
+dealer1 = Dealer()
+
+
+dealer1.dealCards(2, [player1], deck1)
+dealer1.dealCards(3, [player2], deck1)
+dealer1.dealCards(4, [player3], deck1)
+dealer1.printAllPlayerCards_test(players2)
+
+"""
+
+# SPRINT 3 CODE
+def findCardMatch():
```

Comment:
the description for the task was a little vague, but the function is a lot simpler

it takes two cards and returns True or False if it the card.values matches (you don't need to worry about suit) 

so you could do something like:


`
def findCardMatch(Card1, Card2):
        if Card1.value == Card2.value:
            return True
        else:
            return False
`